Microsoft.Azure.Cosmos/src/HttpClient/HttpTimeoutPolicy.cs

```diff
@@ -18,21 +18,45 @@ internal abstract class HttpTimeoutPolicy
 
         public abstract bool ShouldRetryBasedOnResponse(HttpMethod requestHttpMethod, HttpResponseMessage responseMessage);
 
+        public virtual bool ShouldThrow503OnTimeout => false;
+
         public static HttpTimeoutPolicy GetTimeoutPolicy(
            DocumentServiceRequest documentServiceRequest)
         {
+            //Query Plan Requests
             if (documentServiceRequest.ResourceType == ResourceType.Document
                 && documentServiceRequest.OperationType == OperationType.QueryPlan)
             {
-                return HttpTimeoutPolicyControlPlaneRetriableHotPath.Instance;
+                return HttpTimeoutPolicyControlPlaneRetriableHotPath.InstanceShouldThrow503OnTimeout;
             }
 
+            //Partition Key Requests
             if (documentServiceRequest.ResourceType == ResourceType.PartitionKeyRange)
             {
                 return HttpTimeoutPolicyControlPlaneRetriableHotPath.Instance;
             }
 
+            //Data Plane Read & Write
+            if (!StaticIsMetaData(documentServiceRequest))
+            {
+                return HttpTimeoutPolicyDefault.InstanceShouldThrow503OnTimeout;
+            }
+
+            //Meta Data Read
+            if (StaticIsMetaData(documentServiceRequest) && documentServiceRequest.IsReadOnlyRequest)
+            {
+                return HttpTimeoutPolicyDefault.InstanceShouldThrow503OnTimeout;
+            }
+
+            //Default behavior
             return HttpTimeoutPolicyDefault.Instance;
+
+            static bool StaticIsMetaData(DocumentServiceRequest request)
```

Comment:
IsMetaData is ok as a name

```suggestion
            static bool IsMetaData(DocumentServiceRequest request)
```